joomla/joomla-cms
# [4.0] 29957 css

Pull Request for Issue #29957

### Summary of Changes

icons now center properly

### Testing Instructions
go to template styles and duplicate the atum template.
verify gold star and gray star are vertically aligned.
NPM CI REQUIRED!


### Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/400092/86511849-d3b8e080-bdf4-11ea-83ff-a2eedd8b98c4.png)

### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/1850089/86530965-05788880-be83-11ea-81b3-f435a738a71c.png)

### Documentation Changes Required
none
